frontend/src/entities/group/detail/group-detail.actions.tsx

```diff
@@ -107,7 +106,7 @@ const GroupActionHandler = (
                 setDeleteModal({
                     resourceName: 'Group',
                     resourceType: 'group',
-                    postConfirm: () => dispatch(getAllGroups()),
+                    postConfirm: () => dispatch(getAllGroups(window.location.href.includes('admin'))),
```

Comment:
What if some mad lad creates a group with the name `admin` in it?

Reply:
Good call...